### ISSUE TITLE
On VR mode the whole view becomes pink.

### ISSUE THREAD

**Issue by nooranihasnain** [De-Panther/unity-webxr-export#412]
**Describe the bug**
I added the WebXR project and followed all steps to import the package to an existing Unity project. On an empty project the sample scene works fine but on the existing project that I'm working on, whenever I switch to VR Mode the whole view becomes Pink.

**To Reproduce**
1. Import the Packages as provided in the documentations through openUPM.
2. Import samples and add the sample Built in URP scene in build settings.
3. Build the project and run it on Oculus Quest 2.

**Expected behavior**
The VR mode should run the VR controls and demo on the build.

**Unity info (please complete the following information):**
- Editor version: 2022.3.15
- WebXR package version: 0.22.1
- WebXR Interactions version: 0.22.0
- Git or OpenUPM: openUPM
- Using old components (like the Desert sample) or Input System + XR Interaction Toolkit: Using Built in render pipeline sample
- Built-in render pipeline or URP: Built in Render pipeline

**Headset (please complete the following information):**
 - Device: Meta Quest 2
 - OS: Meta Quest build 68.0
 - Browser: Meta Quest Browser 34.4

**Comment by De-Panther** [comment]
Can you please share your project in a repository for test?

**Comment by nooranihasnain** [comment]
It seems like a bug. I'm unable to share the project because it's a client project but I figured out the issue.
The empty project I was testing, I had imported the URP version of the sample and just changed the materials to Standard shader which worked fine for me.
In the existing project, I imported the built in sample scene which wasn't working for me.

**Solution: **
Imported the URP sample and changed the materials to standard.
